### PR TITLE
Update flowers accepted by bees, simplify schema language, add new flowers

### LINKED ISSUE #4: Bee flowers_accepted should be array rather than string

### PR DESCRIPTION
**Pull request by carl5004**
Working / draft pull request to update flowers accepted by bees. Most recent commit closes #4

### CODE REVIEW

**Comment by floogulinc** [comment]
I feel like new flowers should be a separate pull request.

**Comment by carl5004** [comment]
> I feel like new flowers should be a separate pull request.

KK suggested that I add them here. 
Is it possible to move a commit? - If it is, then I could do that if you feel it's really necessary.

**Comment by floogulinc** [comment]
> > I feel like new flowers should be a separate pull request.
> 
> KK suggested that I add them here.
> Is it possible to move a commit? - If it is, then I could do that if you feel it's really necessary.

It probably wouldn't be practical to split them out at this point so I think it's fine to do those here.

File: bees.schema.json [hunk 1]
```diff
@@ -20,18 +20,21 @@
             "type": "string"
           },
           "flowers_accepted": {
-            "type": "string"
+            "type": "array"
```
Comment (floogulinc) [inline bees.schema.json:23]: The `flowers_accepted` array should only contain strings and they should be unique.
```suggestion
            "type": "array",
            "uniqueItems": true,
            "items": {
            	"type": "string"
            }
```

File: flowers.schema.json [hunk 2]
```diff
@@ -19,16 +19,19 @@
           "sci_name": {
             "type": "string"
           },
-          "image_file": {
-            "type": "string"
+          "art_file": {
```
Comment (kklamberty) [inline flowers.schema.json:22]: Will there always be just one art file? I think that seems sensible, but wanted to verify.

File: bees.schema.json [hunk 3]
```diff
@@ -75,16 +81,16 @@
               "pollen_collection": {
                 "type": "string"
               },
-              "when_are_they_active": {
+              "activity": {
```
[the remark this answers is not in the document]
Comment (carl5004) [reply bees.schema.json:88]: I think I'll change it to "when_active". I was trying to differentiate between the active period (month and day) versus the description of when they are active.

[the remark this answers is not in the document]
Comment (kklamberty) [reply bees.schema.json:88]: I think I didn't pay enough attention to the context here. It seems like `activity` is probably just fine given that it is the text description of their activity during the `active_period`.

[the remark this answers is not in the document]
Comment (carl5004) [reply bees.schema.json:88]: Okay. I will leave it with `activity` since that's the way it is now.

**Pull request merged**